config/config.toml.dist

```diff
@@ -111,8 +111,15 @@ chartVersion = ""
 imageTag = "0.1.2"
 
 [servicemesh]
-istioOperatorChartVersion = "0.0.3"
+istioOperatorChartVersion = "0.0.9"
 grafanaDashboardLocation = "https://raw.githubusercontent.com/banzaicloud/banzai-charts/master/istio/deps/grafana/dashboards"
+istioOperatorChartName = "istio-operator"
+istioPilotImage = "banzaicloud/istio-pilot:1.1.4-bzc"
+istioMixerImage = "banzaicloud/istio-mixer:1.1.4-bzc"
+
+[uistio]
+chartVersion = "0.0.2"
```

Comment:
```suggestion
chartVersion = "0.0.4"
```
0.0.4 is already used here: https://github.com/banzaicloud/pipeline/blob/9c8d8d8a0bab730d592e2c580bb464a833b12f94/config/configuration.go#L302

(We will need to update it soon anyway.)